htdocs/compta/stats/cabyprodserv.php

```diff
@@ -486,6 +490,9 @@
 	print '<br>';
 	print img_picto('', 'company', 'class="pictofixedwidth"');
 	print $form->select_company($selected_soc, 'search_soc', '', $langs->trans("ThirdParty"));
+	print '&nbsp; &nbsp;';
+	print $langs->trans("Commercial").': ';
+	print $form->select_dolusers($selected_commercial, 'search_commercial', 1);
```

Comment:
For user interface consistency, a combo list of users must have before it a
`print img_picto('', 'user', 'class="pictofixedwidth"');`
Also the title "Commercial" must be included as placeholder into the combo (see example on the select_company just before).